.github/scripts/trymerge.py

```diff
@@ -1203,6 +1204,17 @@ def merge_into(
         else:
             print("Missing comment ID or PR number, couldn't upload to Rockset")
 
+        # Usually Github will see that the commit has "resolves <pr_num>" in the
+        # commit message and close the PR, but sometimes it doesn't, leading to
+        # confusion.  When it doesn't, we close it manually.
+        time.sleep(60)  # Give Github some time to close the PR
+        manually_close_merged_pr(
+            pr=self,
```

Comment:
Why not pass the whole list here instead? i.e. `manually_close_merged_pr([self, ...additional_merged_prs], ...)`. This would make writing `manually_close_merged_pr` a bit easier.  The other way is to put the 3-step process `message=`, `gh_post_pr_comment`, `gh_close_pr` into a util function so you can just call it instead.